src/components/LeftContainer.js

```diff
@@ -156,6 +174,16 @@ const LeftContainer = ({
         <div className="logger__head_left">
           <h3 className="logger__heading">Editor</h3>
           <div className="tooltipBoundary">
+            <button className="btn" type="button">
+              <img
+                src={`${process.env.PUBLIC_URL}/assets/reset.png`}
+                title="Reset"
+                alt="Reset Code"
+                width="26px"
```

Comment:
@Vipu97 hi we are using images of 24x24 dimension please refer to the icons below